django_rq/templates/django_rq/job_detail.html

```diff
@@ -174,6 +174,12 @@
                 <input type="submit" value="Requeue" class="default" name="requeue">
             </form>
         {% endif %}
+        {% if job.is_deferred  %}
+            <form method = 'POST' action = "{% url 'rq_enqueue_job' queue_index job.id %}">
```

Comment:
I think we should just show this button regardless of the job’s status. Sometimes I have the need to requeue finished jobs so this is a useful option to have.